### DISCUSSION TITLE
CoPilot is

### DISCUSSION THREAD

**Discussion by saram7619** [community/community#109154] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I am having problems with my copilot. 
I am using vscode on a Mac 

Whenever I ask copilot a question in the chatbox, I get this response.
unhandled status from server: 410 {"error":{"agent-version":"","details":"https://github.blog/changelog/label/copilot/#%F0%9F%8C%90-upcoming-deprecation-of-copilot-chat-api-endpoints","editor-plugin-version":"copilot-chat/0.6.0","editor-version":"vscode/1.81.1","engine":"copilot-chat","message":"copilot-proxy.githubusecontent.com is no longer accepting chat requests. Please update your IDE extension to continue using GitHub Copilot chat."}}

I have uninstalled and reinstalled the extension, but I am still getting this message.

How do I fix this error?

**Comment by davecheney** [comment, marked as answer]
@saram7619 newer versions of the extension require newer versions of vscode. vscode insiders is no longer needed, please switch to regular vscode, then install the copilot extension which will install the matching chat extension. Keep clicking update and restart until vscode is happy, then you should be good to go.
